src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -330,4 +330,23 @@ function rhs_parabolic!(du_ode, u_ode, semi::SemidiscretizationHyperbolicParabol
 
     return nothing
 end
+
+function rhs_hyperbolic_parabolic!(du_ode, u_ode, semi::SemidiscretizationHyperbolicParabolic, t)
+    @trixi_timeit timer() "hyperbolic-parabolic rhs!" begin 
+        # Implementation of split ODE problem in OrdinaryDiffEq
+        du_ode_hyp = similar(du_ode)
+        rhs!(du_ode_hyp, u_ode, semi, t)
+        rhs_parabolic!(du_ode, u_ode, semi, t)
+        du_ode .+= du_ode_hyp
```

Comment:
This mimicks 

```julia
function (f::SplitFunction)(du, u, p, t)
    f.f1(f.cache, u, p, t)
    f.f2(du, u, p, t)
    du .+= f.cache
end
```

which resides in [SciMLBase.jl/src/scimlfunctions.jl](https://github.com/SciML/SciMLBase.jl/blob/091c91aa1da1b8f14e1c7bd3e7c439dc6a218784/src/scimlfunctions.jl#L2139-L2143).